coramin/relaxations/auto_relax.py

```diff
@@ -857,12 +857,57 @@ def _relax_expr(expr, aux_var_map, parent_block, relaxation_side_map, counter, d
     return new_expr
 
 
-def relax(model, descend_into=None, in_place=False, use_fbbt=True):
+def relax(model, descend_into=None, in_place=False, use_fbbt=True, fbbt_options=None):
+    """
+    Create a convex relaxation of the model.
+
+    Parameters
+    ----------
+    model: pyomo.core.base.block._BlockData or pyomo.core.base.PyomoModel.ConcreteModel
+        The model or block to be relaxed
+    descend_into: type or tuple of type
+        The types of pyomo components that should be checked for constraints to be relaxed. The
+        default is (Block, Disjunct).
+    in_place: bool
+        If True, then model will be modified in place. If False, model will be cloned, and
+        the clone will be relaxed.
+    use_fbbt: bool
+        If True, then FBBT will be used to tighten variable bounds.
```

Comment:
Can you add the default to this docstring (reading it implies that the default is False, which is not correct).